blog/index.md

```diff
@@ -3,6 +3,17 @@ title: Blog
 layout: blog-index
 ---
 
+# [Deploy Skydive on top of OpenStack thanks to Tripleo](/blog/tripleo.html)
+## by Sylvain Afchain, 07/08/2018
+
+Skydive supports multiple deployment ways, from containers(Kubernetes, OpenShift) to Ansible playbook. In this blogpost I will explain how to
+deploy Skydive on top of OpenStack using Tripleo. Skydive is already integrated as part of the Tripleo deployment since the Queens realase but
```

Comment:
Support for Skydive is already integrated in TripleO since the Queens release but this support has been reworked the new config download](https://docs.openstack.org/tripleo-docs/latest/install/advanced_deployment/ansible_config_download.html) feature.